features/bevragen/persoon/verblijfplaats/adres/verblijfadres/in-onderzoek.feature

```diff
@@ -0,0 +1,143 @@
+
+Rule: Het in onderzoek zijn van een veld en bijbehorende datumIngangOnderzoek worden geleverd als het betreffende veld wordt gevraagd
+
+  Abstract Scenario: '<type>' is in onderzoek en alle verblijfadres velden wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                 | waarde |
+    | gemeentecode (92.10) | 0518   |
+    | straatnaam (11.10)   | Spui   |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | verblijfplaats.verblijfadres    |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                                         | waarde         |
+    | type                                         | Adres          |
+    | inOnderzoek.type                             | <type io>      |
+    | inOnderzoek.datumIngangOnderzoek.type        | <datum type>   |
+    | inOnderzoek.datumIngangOnderzoek.datum       | <datum>        |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | <lang formaat> |
+    En heeft de 'verblijfplaats' de volgende 'verblijfadres' gegevens
+    | naam                                         | waarde                   |
+    | korteNaam                                    | Spui                     |
+    | inOnderzoek.aanduidingBijHuisnummer          | <aanduidingBijHuisnr io> |
+    | inOnderzoek.huisletter                       | <huisletter io>          |
+    | inOnderzoek.huisnummer                       | <huisnr io>              |
+    | inOnderzoek.huisnummertoevoeging             | <huisnrtoevoeging io>    |
+    | inOnderzoek.korteNaam                        | <korteNaam io>           |
+    | inOnderzoek.straat                           | <straat io>              |
+    | inOnderzoek.postcode                         | <postcode io>            |
+    | inOnderzoek.woonplaats                       | <woonplaats io>          |
+    | inOnderzoek.datumIngangOnderzoek.type        | Datum                    |
+    | inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01               |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002              |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | aanduidingBijHuisnr io | huisletter io | huisnr io | huisnrtoevoeging io | straat io | postcode io | korteNaam io | woonplaats io | type io | datum type | datum      | lang formaat | type                          |
```

Comment:
De kolom 'type' heeft alleen in de eerste rij een waarde. De daarop volgende rijden moeten in die kolom achtereenvolgende de waardes 'hele groep adres', 'straatnaam', 'naam openbare ruimte', 'huisnummer', 'huisletter', 'huisnummertoevoeging', 'aanduiding bij huisnummer', 'postcode' en 'woonplaatsnaam' krijgen.